docs/using/recipes.md

```diff
@@ -19,6 +19,33 @@ Then build a new image.
 docker build --rm -t jupyter/my-datascience-notebook .
 ```
 
+To use a requirements.txt file, first create your `requirements.txt` file
+with the listing of packages desired.  Next, create a new Dockerfile like the one shown below.
+
+```dockerfile
+# Start from a core stack version
+FROM jupyter/datascience-notebook:9f9e5ca8fe5a
+# Install from requirements.txt file
+COPY requirements.txt /tmp/
+RUN pip install --requirement /tmp/requirements.txt && \
+    fix-permissions $CONDA_DIR && \
+    fix-permissions /home/$NB_USER
+COPY . /tmp/
```

Comment:
It might be good to add a comment about what this extra copy is doing in both cases. I'm guessing its to add additional project files?

Reply:
Yes, the extra copy (COPY . /tip/ ) is not needed.  It was there for adding additional project files in my setup.